extensions/3DTILES_implicit_tiling/schema/subtree/contentMetadata.schema.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Do we forsee that `contentMetadata` and tileMetadata` will diverge? the schema files are pretty much identical. Might be okay, just thought I'd ask.

Reply:
Unlikely that they would diverge, but I think two files is ok for now.